x/pki/client/cli/query_approved_certificates.go

```diff
@@ -0,0 +1,82 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/x/pki/types"
+)
+
+func CmdListApprovedCertificates() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "all-x509-certs",
+		Short: "Gets all certificates (root, intermediate and leaf)",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx := client.GetClientContextFromCmd(cmd)
```

Comment:
does the command respect configuration (e.g. output=json or output=text) in client.toml?